src/main/java/seedu/address/logic/parser/ExportCommandParser.java

```diff
@@ -0,0 +1,28 @@
+package seedu.address.logic.parser;
+
+import java.io.File;
+
+import seedu.address.logic.commands.ExportCommand;
+import seedu.address.logic.parser.exceptions.ParseException;
+
+/**
+ * Parses input arguments and creates a new ExportCommand object.
+ */
+public class ExportCommandParser implements Parser<ExportCommand> {
+
+    /**
+     * Parses the given argument {@code String} in the context of the ExportCommand
+     * and returns an ExportCommand object for execution.
+     * @throws ParseException if the user input does not conform the expected format
+     */
+    public ExportCommand parse(String args) throws ParseException {
+        try {
+            File file = ParserUtil.parseFile(args);
+            return new ExportCommand(file);
+        } catch (ParseException pe) {
+            throw new ParseException(pe.getMessage());
+        }
+    }
+    // TODO: Combine ImportCommandParser and ExportCommandParser
```

Comment:
Would be good if the team could make use of the issue tracker to keep track of such TODO items to be implemented as well